bravado/response.py

```diff
@@ -69,9 +69,10 @@ def result(self, **kwargs):
 
         if self.cancelled():
             raise CancelledError()
-        response = self._request.wait(timeout=timeout)
+        response = self._request.result(timeout=timeout)
         try:
-            response.raise_for_status()
+            if hasattr(response, 'raise_for_status'):
```

Comment:
I am not sure if this is the correct way. `requests` silently stores errors when making the api call whereas fido throws up the exception during the api call. The `raise_for_status` is specific to `requests` to make it throw the exception if any.